tests/utils/test_odcs.py

```diff
@@ -202,3 +202,19 @@ def assert_request_token(request, session):
     if ODCSClient.OIDC_TOKEN_HEADER in session.headers:
         expected_token = session.headers[ODCSClient.OIDC_TOKEN_HEADER]
     assert request.headers.get(ODCSClient.OIDC_TOKEN_HEADER) == expected_token
+
+
+@responses.activate
+def test_wait_for_compose_timeout():
+    compose_url = '{}composes/{}'.format(ODCS_URL, COMPOSE_ID)
+
+    responses.add(responses.GET, compose_url, body=compose_json(0, 'generating'))
+    (flexmock(time)
+        .should_receive('time')
+        .and_return(2000, 1000)
```

Comment:
What are the calls to time.time() other than for fetching the start and end times here? It is a bit confusing, since the 3rd call should return `2000` as well. A comment here could help.

Reply:
@athos-ribeiro Updated. PTAL.